interfaces/sourcegen/src/sourcegen/headers/ctthermo_auto.yaml

```diff
@@ -46,7 +46,11 @@ recipes:
 - name: elementName  # Renamed in Cantera 3.2 (previously getElementName)
 - name: speciesName  # Renamed in Cantera 3.2 (previously getSpeciesName)
 - name: elementIndex
+  # alternate version to be removed after Cantera 3.2
+  wraps: elementIndex(const string&, bool)
 - name: speciesIndex
+  # alternate version to be removed after Cantera 3.2
+  wraps: speciesIndex(const string&, bool)
```

Comment:
This additional, required argument seems a little strange in the case of Clib, where you have to check whether the return value is valid no matter what.

Reply:
It is unfortunately required here, as I'd like to reach the new version (which doesn't have a default argument yet).